### ISSUE TITLE
Telegram Command Pallet causing crashes when description field is empty  - Potentially an easy fix

### ISSUE THREAD

**Issue by Macrofarad** [guidone/node-red-contrib-chatbot#551]
Error log (docker), 
Node red 3.0.0

20 Jul 17:30:03 - [error] Error: ETELEGRAM: 400 Bad Request: command description must be non-empty

Once I put text into the description field it seemed to fix the crash issue. So you may be able to fix it by just inserting a dummy text upon no user text provided.

**Comment by guidone** [comment]
Thanks for the feedback, I'll include in the next release.

Btw, did you have the chance to give a look to Mission Control?

**Comment by Macrofarad** [comment]
I actually set it up last night, gotta say, I was impressed. Frankly, this morning I actually woke up two hours early so just I could play around with the new changes a bit more before I had to log into work.

The core itself seems pretty solid and the direction is great for sure, that said, there's a lot of areas where it could use usability improvements or better tooltips for the sake of clarity.   There are also a few areas like the welcome message plugin that are semi-unfinished; as of last night (I'm GMT-7 for timezone context) the subflow it would suggest for users was still a store open/close hours setup that didn't quite work out of the box.  Still, fantastic job, the effort you've put in shows.

If you have a preferred area to put feedback, I can shoot you a list of what I spotted.  Additionally, I use node red both for large scale automation at work as well as to manage my house so I'd happy to help spot bugs or point out potential usability improvements if you wanted.
